modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for SHSimpleIDListFromPath
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "shelltest.h"
+#include <shellutils.h>
+
+enum { DIRBIT = 1, FILEBIT = 2 };
+
+static int FileStruct_Att(LPCITEMIDLIST pidl)
+{
+    return pidl && pidl->mkid.cb > 14 ? *((WORD*)((BYTE*)(pidl) + 12)) : (1 << 31); // See FileStruct in pidl.h
+}
+
+START_TEST(SHSimpleIDListFromPath)
+{
+    HRESULT hr;
+    WCHAR szPath[MAX_PATH];
+    GetWindowsDirectoryW(szPath, _countof(szPath));
+    LPITEMIDLIST item;
```

Comment:
Since we are in C++, you could move this declaration a little bit later when it starts to be used.